sphinx_gallery/gen_rst.py

```diff
@@ -505,16 +530,16 @@ def execute_script(code_block, example_globals, image_path, fig_count,
         fig_count += 1  # raise count to avoid overwriting image
 
         # Breaks build on first example error
-
-        if gallery_conf['abort_on_example_error']:
+        # Eventually this should hopefully be unified with the default conf
+        if gallery_conf.get('abort_on_example_error', True):
```

Comment:
I was hitting a problem where if an example raised an error at a certain point, *this* line then raised an error during handling (because the value didn't exist) so it never got to the proper `raise` line below. Again an orthogonal change I guess but I needed it to get things working properly during testing.